src/serlo-editor-repo/plugin-text/index.tsx

```diff
@@ -35,6 +36,17 @@ const createTextPlugin = (
       return value
     },
     deserialize(value) {
+      // If the first child of the Element is an empty object,
+      // replace it with an empty Text node
+      // https://docs.slatejs.org/concepts/11-normalizing#built-in-constraints
+      const firstChild = (value[0] as CustomElement).children[0]
+      if (isEmptyObject(firstChild)) {
+        return {
+          value: [{ children: [{ text: '' }] } as CustomElement],
```

Comment:
@hejtful What happens if value has more than one element? Will this case not happen? Because it looks like all remaining children are discarded here.

Reply:
@Entkenntnis Slate automatically adds this empty text node in children in the new version, so the new articles are safe.

The condition on line 43 will only be true in cases where an empty text editor was saved with the old version, therefore we don't expect any more children.

Or do you see something I missed?